### PR TITLE
chore: updating Babel, Storybook and other deps

### PR DESCRIPTION
**Pull request by helendurranttw**
## 🖼 Context

We had some security issues from dependabot, and rather then work around them I thought I'd update everything.

## 🚀 Changes

Updated deps and did any required migration of associated config.

### CODE REVIEW

File: packages/components/.storybook/main.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
 
+const stories = ['../src/**/*.story.js'];
+
+if (process.env.STORYBOOK_CIRCLECI) {
+  stories.push('../test/**/*.story.js');
+}
+
 // Export a function. Accept the base config as the only param.
 module.exports = {
+  stories: ['../src/**/*.story.js'],
```
Comment (andreapiras) [inline packages/components/.storybook/main.js:11]: 😍😍😍😍

File: packages/components/package.json [hunk 2]
```diff
@@ -67,26 +67,26 @@
     "chromatic": "^5.4.0",
     "html-loader": "^1.3.2",
     "lodash.times": "^4.3.2",
-    "rollup": "^2.7.6",
-    "rollup-plugin-babel": "^4.4.0",
-    "rollup-plugin-postcss": "^2.0.3",
+    "rollup": "^2.34.0",
```
Comment (andreapiras) [inline packages/components/package.json:70]: 😍

File: packages/components/package.json [hunk 3]
```diff
@@ -42,22 +42,22 @@
...
-    "@storybook/addon-actions": "^5.3.21",
-    "@storybook/addon-info": "^5.3.21",
-    "@storybook/addon-knobs": "^5.3.21",
-    "@storybook/react": "^5.3.21",
-    "@transferwise/icons": "^2.5.0",
+    "@storybook/addon-a11y": "^6.1.9",
+    "@storybook/addon-actions": "^6.1.9",
+    "@storybook/addon-docs": "^6.1.9",
+    "@storybook/addon-knobs": "^6.1.9",
+    "@storybook/react": "^6.1.9",
```
Comment (andreapiras) [inline packages/components/package.json:59]: 😍

**Review by andreapiras — APPROVED** [review]

File: packages/components/.storybook/preview.js [hunk 4]
```diff
@@ -1,68 +1,48 @@
...
-  padding: '80px',
-  minHeight: '100vh',
-  width: '100vw',
-};
-
+// Set up Axe to run in the console
 if (process.env.NODE_ENV !== 'production') {
   const ReactDOM = require('react-dom');
-  const axe = require('react-axe');
+  const axe = require('@axe-core/react');
```
Comment (dozortsev) [inline packages/components/.storybook/preview.js:13]: wonder whether all this updates for Storybook and in particular for react axe library would fix annoying issue which we have on all stories https://transferwise.atlassian.net/browse/DS-149, have you checked it? 🙂

File: packages/components/package.json [hunk 5]
```diff
@@ -67,26 +67,26 @@
...
+    "rollup": "^2.34.0",
+    "@rollup/plugin-babel": "^5.2.2",
+    "rollup-plugin-postcss": "^2.8.2",
     "rollup-plugin-uglify": "^6.0.4"
   },
   "peerDependencies": {
-    "@transferwise/icons": "^2.5.0",
-    "@transferwise/neptune-css": "^4.0.8",
+    "@transferwise/icons": "^2.6.0",
+    "@transferwise/neptune-css": "^5.0.1",
```
Comment (dozortsev) [inline packages/components/package.json:77]: shouldn't `@transferwise/neptune-css` get bumped automatically by Lerna? 🤔

File: packages/components/rollup.config.js [hunk 6]
```diff
@@ -28,14 +28,15 @@ const globals = {
   react: 'React',
   'react-dom': 'ReactDOM',
   'prop-types': 'PropTypes',
+  icons: '@transferwise/icons',
```
Comment (dozortsev) [inline packages/components/rollup.config.js:31]: what this will do?

**Review by dozortsev — APPROVED** [review]

File: packages/components/package.json (hunk 5 above)
Comment (helendurranttw) [reply packages/components/package.json:77]: Yeah it should, this one was out of sync with the others - not sure why 😟

File: packages/components/rollup.config.js (hunk 6 above)
Comment (helendurranttw) [reply packages/components/rollup.config.js:31]: In the umd bundle you have a reference to 'icons' when using icons inside neptune web - this tells the bundle which package provides this global

**Pull request merged**